images/macos/toolsets/toolset-15.json

```diff
@@ -36,12 +36,12 @@
         ],
         "addons": [],
         "additional_tools": [
-            "cmake;3.22.1"
+            "cmake;3.30.2"
```

Comment:
Shouldn't work as I understand. They use prebaked cmake releases and 3.22.1 is the latest available.